src/components/transport_manager/include/transport_manager/usb/libusb/usb_handler.h

```diff
@@ -97,11 +101,19 @@ class UsbHandler {
   libusb_hotplug_callback_handle left_callback_handle_;
 
   friend void* UsbHandlerThread(void* data);
-  friend int ArrivedCallback(libusb_context* context, libusb_device* device,
+  friend int 
```

Comment:
It will be more readable if change code block to:
```
#ifdef OS_WIN32
friend int LIBUSB_CALL ArrivedCallback(libusb_context* context, libusb_device* device,
                                                                    libusb_hotplug_event event, void* data);
friend int LIBUSB_CALL LeftCallback(libusb_context* context, libusb_device* device,
                                                               libusb_hotplug_event event, void* data);
friend void* UsbHotPlugThread(void* data);
void UsbThread();
bool IsUsbEqual(libusb_device *devd,libusb_device *devs);
#else
  ArrivedCallback(libusb_context* context, libusb_device* device,
                               libusb_hotplug_event event, void* data);
  LeftCallback(libusb_context* context, libusb_device* device,
                          libusb_hotplug_event event, void* data);
```